Source/Venice/Channel/Channel.swift

```diff
@@ -67,26 +67,23 @@ public final class Channel<T>: Sequence {
     }
 
     /// Closes the channel. When a channel is closed it cannot receive values anymore.
-    public func close() -> Bool {
-        if closed {
-            return false
-        }
+    public func close() throws {
+        guard !closed else { return }
```

Comment:
Does it make sense to make it throw? Or rather, do we get any value in making `Channel<T>` adopt `Closable` protocol? Let's think about usage. Could be inconvenient to have it throw.